src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <label>
+      <form
+        class="inputForm top"
+        action="search"
+        method="get"
+        data-qa="big"
+      >
+        <img
+          class="big_img"
+          src="images/Search.svg"
+          alt="img">
```

Comment:
check out point 5 in the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
```suggestion
        <img
          class="big_img"
          src="images/Search.svg"
          alt="img"
        >
        
```